pkg/zonemgr/zone_file_cache.go

```diff
@@ -27,27 +27,27 @@ const (
 var soaSerialReg = regexp.MustCompile("SOA .*\\(([0-9]+) ")
 
 type ZoneFileCache struct {
-	soaSerial          int
+	SoaSerial          int
 	adminEmail         string
-	nameServerName     string
-	nameServerIP       string
-	domain             string
+	NameServerName     string
```

Comment:
Why is it required to export the members?

Reply:
I use it in unit tests, for example here: https://github.com/kubevirt/kubesecondarydns/pull/51/files#diff-225df37884a11dd32c318ab3d1a4adc505eb22e40d8f2f8cbd93d7fb5ca06d9cR37

Reply:
it is the same package, it can basically be unexported and still be used in the unit tests
this is the right thing imo

Reply:
I moved ZoneFile and ZoneFileCache to internal/ package so I can't use anything from the inside unless it's Public.
I left test files outside internal/ package on purpose to make sure I use only Public resources as Alona suggested

Reply:
I think unit tests should be inside internal, unless they are testing API and then they should be outside